experimental/CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ For notes on migrating to 2.x / 0.200.x see [the upgrade guide](doc/upgrade-to-2
 
 * fix(otlp-exporter-base): prioritize `esnext` export condition as it is more specific [#5458](https://github.com/open-telemetry/opentelemetry-js/pull/5458)
 * fix(instrumentation-fetch): Use ESM version of semconv instead of CJS. Users expecting mixed ESM and CJS modules will now only get ESM modules. [#5878](https://github.com/open-telemetry/opentelemetry-js/pull/5878) @overbalance
+* fix(instrumentation-grpc): Use ESM version of semconv instead of CJS. Users expecting mixed ESM and CJS modules will now only get ESM modules. [#5884](https://github.com/open-telemetry/opentelemetry-js/pull/5884) @overbalance
```

Comment:
I think we can move this entry to internal as it's a refactor ("changed internal import paths"). The `@opentelemetry/instrumentation-grpc` package only contains CJS at this point - so it would not change anything right now.